test/e2e/storage/testsuites/multivolume.go

```diff
@@ -429,6 +429,10 @@ func (t *multiVolumeTestSuite) DefineTests(driver storageframework.TestDriver, p
 			e2eskipper.Skipf("Driver %q does not support multiple concurrent pods - skipping", dInfo.Name)
 		}
 
+		if l.driver.GetDriverInfo().Name == "vsphere" {
+			e2eskipper.Skipf("Driver %q does not support read only volumes - skipping", dInfo.Name)
+		}
```

Comment:
it has to be read-only raw block volumes right? So we also need to check for `pattern.VolMode` ?

Reply:
the test case (should concurrently access the single read-only volume from pods on the same node) creates readonly volume
https://github.com/kubernetes/kubernetes/blob/a1128e380c2cf1c2d7443694673d9f1dd63eb518/test/e2e/storage/testsuites/multivolume.go#L442

so, I don't think we need to check for `pattern.VolMode`